docs/docs/guides/cli/how-to-automatically-deploy.mdx

```diff
@@ -0,0 +1,57 @@
+
+# How to automatically deploy 
```

Comment:
```suggestion
# How to automatically sync your change in Lightdash
```

I'm not sure if we should also add `automatically deploy` somewhere in the title too so it's more searchable? 

I just think as a data person `automatically deploy` isn't a word I'd be looking for (it's more "syncing" that I'd be trying to find)